openreview/conference/templates/areachairWebfield.js

```diff
@@ -651,12 +672,94 @@ var registerEventHandlers = function() {
   });
 
   $('#group-container').on('click', 'a.collapse-btn', function(e) {
-    $(this).next().slideToggle();
+    // $(this).next().slideToggle();
     if ($(this).text() === 'Show reviewers') {
       $(this).text('Hide reviewers');
     } else {
       $(this).text('Show reviewers');
     }
+    // return false;
+  });
+
+  $('#group-container').on('click', 'button.btn.assign-reviewer-button', function(e) {
+    var $link = $(this);
+    var paperNumber = $link.data('paperNumber');
+    var paperForum = $link.data('paperForum');
+    console.log(paperNumber);
+    console.log(paperForum);
+    var $currDiv = $('#' + paperForum + '-add-reviewer');
+    console.log($currDiv);
+    userToAdd = $currDiv.find('input').val().trim();
+    console.log(userToAdd);
+    nextAnonNumber = findNextAnonGroupNumber(paperNumber);
+    console.log(nextAnonNumber);
+    $.when(
+      Webfield.put('/groups/members', {
+        id: CONFERENCE_ID + '/Paper' + paperNumber + '/Reviewers',
+        members: [userToAdd]
+      }),
+      Webfield.post('/groups', {
+        id: CONFERENCE_ID + '/Paper' + paperNumber + '/AnonReviewer' + nextAnonNumber,
+        members: [userToAdd],
+        readers: [
+          CONFERENCE_ID + '/Program_Chairs',
+          CONFERENCE_ID + '/Paper' + paperNumber + '/Area_Chairs',
+          CONFERENCE_ID + '/Paper' + paperNumber + '/AnonReviewer' + nextAnonNumber],
+        writers: [
+          CONFERENCE_ID + '/Program_Chairs',
+          CONFERENCE_ID + '/Paper' + paperNumber + '/Area_Chairs'],
+        signatures: [CONFERENCE_ID + '/Paper' + paperNumber + '/Area_Chairs'],
+        signatories: [CONFERENCE_ID + '/Paper' + paperNumber + '/AnonReviewer' + nextAnonNumber]
+      }),
+      promptMessage('Reviewer ' + view.prettyId(userToAdd) + ' has been assigned to paper ' + paperNumber)
+    );
+    var forumUrl = 'https://openreview.net/forum?' + $.param({
+        id: paperForum,
+        noteId: paperForum,
+        invitationId: CONFERENCE_ID + '/-/Paper' + paperNumber + '/Official_Review'
+      });
+    var lastReminderSent = localStorage.getItem(forumUrl + '|' + userToAdd);
```

Comment:
if the user is new why do we need to get the last reminder?

Reply:
This was mostly because I was not sure of the return for Storage.getItem() in case the item is missing. I have replaced it with null now.